siliconcompiler/tools/klayout/klayout.py

```diff
@@ -78,7 +78,9 @@ def setup(chip, mode="batch"):
     else:
         klayout_exe = 'klayout'
 
-    if mode == 'show':
+    is_show = mode == 'show' or step == 'show'
```

Comment:
This is an example of a place where having separate task definitions for klayout would avid confusion. Otherwise it looks fine.

Reply:
yes, that will simplify this a lot, there are some confusing if trees now.